# HG changeset patch
# User Roland Haas <roland.haas@physics.gatech.edu>
# Date 1315525058 14400
# Node ID 7f69434460672283638d6e024fa7e97d8ee7afcd
# Parent  d87fce06a3cde72fae58b401604e6d4e1e007cdd
Carpet: define _GNU_SOURCE to make sched_getaffinity prototype visible

diff --git a/Carpet/Carpet/src/SetupGH.cc b/Carpet/Carpet/src/SetupGH.cc
--- a/Carpet/Carpet/src/SetupGH.cc
+++ b/Carpet/Carpet/src/SetupGH.cc
@@ -1,3 +1,5 @@
+#define _GNU_SOURCE 1 // needed for sched_getaffinity, best at the top to avoid inconsistent includes
+
 #include <cassert>
 #include <climits>
 #include <cmath>
